Lib/functools.py

```diff
@@ -234,9 +234,9 @@ def __ge__(self, other):
 
 _initial_missing = object()
 
-def reduce(function, sequence, initial=_initial_missing):
+def reduce(function, sequence, /, initial=_initial_missing):
```

Comment:
This breaking backward compatibility, see https://github.com/python/cpython/pull/121677#discussion_r1676808256 for more information.

Reply:
Thanks for letting me know. I will revert this.

On the whole, I am +0 on enabling keyword support for all parameters in `reduce`. Personally I have not felt the need for other parameters barring `initial`. I can see the desire for parity with python version, but I don't see much readability gain from having `function` and `sequence` as keywords, as it is less confusing. Then again, I am generally against position only params for functions.

Reply:
I will change the actual signature. The docstring for python version of `reduce` shows the signature as positional only. Is it okay if I make the docstring signature to this?:
```diff
"""
-    reduce(function, iterable[, initial], /) -> value
+    reduce(function, iterable, /, initial = None) -> value
```
until there is agreement on whether to allow keywords for all params.

Reply:
@sayandipdutta, you shouldn't use ``None`` as default value.

And, I think, there is no need to change sphinx docs further.  ``reduce(function, iterable, /[, initial]) -> value`` - just fine.

If you are going with AC version - see https://github.com/python/cpython/pull/125917#issuecomment-2434922118

Reply:
Ah, I just pushed. Okay, when everything else is sorted, I will fix this at the end.